File: 프로그래머스/0/120829. 각도기/각도기.java
class Solution {
    public int solution(int angle) {
        int answer = 0;
        int a = 4;
        int b = 3;
        int c = 2;
        int d = 1;
        
        if (angle==180) {
        answer=a;
        }
        
        else if (angle==90) {
        answer=c;
        }
        
        else if (angle<90){
         answer=d;
        }    
        else if (90<angle){
         answer=b;
        } 
        return answer;
    }
}

File: C++/프로그래머스/0/120829. 각도기/각도기.cpp
#include <string>
#include <vector>

using namespace std;

int solution(int angle) {
    int answer = 0;
    if(0<angle && angle<90) answer=1;
    if(90<angle && angle<180) answer=3;
    if(angle==90) answer=2;
    if(angle==180) answer=4;
    return answer;
}